### DISCUSSION TITLE
tendermint -> tendermint-go

### DISCUSSION THREAD

**Discussion by tac0turtle** [tendermint/tendermint#5903] [Ideas]
The Tendermint repo used to be the only implementation of the spec of Tendermint (🤔). But as the work behind tendermint-rs nears a full client it will not be the only implementation. At that time, or before I would propose we rename this repo to tendermint-go. This will be a largely breaking change but is more informative.

**Comment by robert-zaremba** [comment]
This will break imports, unless there will be an active redirection.

**Reply by tac0turtle** [reply]
If the go.mod is kept to `github.com/tendermint/tendermint` there would be no change to existing repos. This is meant towards to the name of the repo. Sorry for confusion
